lib/get.js

```diff
@@ -64,7 +64,8 @@ function getClientSettings() {
   module.exports.requestClient ?
     module.exports.requestClient :
     needle.defaults({
-      response_timeout: 4000
+      response_timeout: 4000,
+      follow_max: 1
```

Comment:
What does `follow_max` do?

Reply:
Oh is this the at most one redirect?

Reply:
yeah, so if someone uses a shortened url to request a marker, the request won't automatically fail because of the redirect.